dist/add-markers.js

```diff
@@ -428,7 +428,8 @@ Fliplet.InteractiveMap.component('add-markers', {
       });
     },
     toggleEditMarkerOverlay: function toggleEditMarkerOverlay() {
-      this.showEditMarkerOverlay = !!!this.showEditMarkerOverlay;
+      this.showEditMarkerOverlay = !this.showEditMarkerOverlay;
```

Comment:
Please update source file instead and get this compiled as described here https://github.com/Fliplet/fliplet-widget-interactive-map#fliplet-interactive-map

Reply:
@squallstar Fixed. And also updated this PR https://github.com/Fliplet/fliplet-widget-interactive-map/pull/52 regarding of what was said.